### PR TITLE
feat(cat-voices): check seed phrase and result panel

### PR DESCRIPTION
**Pull request by damian-molinski**
# Description

This PR adds checking seed phrase order by user as well as result panel

## Related Issue(s)

Part of #816  

## Screenshots

Screenshot 2024-10-01 at 14 34 23
Screenshot 2024-10-01 at 14 34 29

## Please confirm the following checks

* [x] My code follows the style guidelines of this project
* [x] I have performed a self-review of my code
* [x] I have commented my code, particularly in hard-to-understand areas
* [x] I have made corresponding changes to the documentation
* [x] My changes generate no new warnings
* [x] I have added tests that prove my fix is effective or that my feature works
* [x] New and existing unit tests pass locally with my changes
* [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: catalyst_voices/lib/widgets/seed_phrase/seed_phrases_picker.dart [hunk 1]
```diff
@@ -14,17 +15,25 @@ class SeedPhrasesPicker extends StatelessWidget {
   final List<String> words;
 
   /// A set of currently selected seed phrases. Defaults to an empty set.
```
Comment (dtscalac) [inline catalyst_voices/lib/widgets/seed_phrase/seed_phrases_picker.dart:17]:
```suggestion
  /// A list of currently selected seed phrases. Defaults to empty list.
```

**Review by dtscalac — DISMISSED** [review]
LGTM

File: catalyst_voices/lib/pages/registration/create_keychain/stage/seed_phrase_check_panel.dart [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+          ),
+        ),
+        const SizedBox(height: 10),
+        _Navigation(isNextEnabled: _isStageValid),
+      ],
+    );
+  }
+
+  Future<void> _uploadSeedPhrase() async {
+    //
```
Comment (digitalheartxs) [inline catalyst_voices/lib/pages/registration/create_keychain/stage/seed_phrase_check_panel.dart:77]: TODO annotation?

**Review by digitalheartxs — DISMISSED** [review]

**Review by dtscalac — APPROVED** [review]
LGTM

**Pull request merged**